svc-managers/mgrmodel/manager.go

```diff
@@ -161,7 +161,7 @@ func GetManagerByURL(url string) (string, *errors.Error) {
 }
 
 // UpdateManagersData will modify the current details to given changes
-func UpdateManagersData(key string, managerData map[string]interface{}) error {
+func UpdateManagersData(key string, managerData map[string]interface{}, table string) error {
```

Comment:
We don't need to pass the table name, as the method is for updating Managers data.

Reply:
Same function is used for other table updates under manager api

Reply:
Addressed comments

Reply:
But still the param name is managerData.

Reply:
It is changed, here it is showing the outdated info, kindly check in the file

Reply:
This looks outdated, please check in the file